### PR TITLE
Test that Number.prototype.toFixed.length is 1

### PR DESCRIPTION
**Pull request by mzgoddard**
Related: #825 

Add a test to confirm that the length member of Number.prototype.toFixed is 1.

### CODE REVIEW

File: test/built-ins/Number/prototype/toFixed/length.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2017 Michael "Z" Goddard. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.1.3.3
```
Comment (leobalter) [inline test/built-ins/Number/prototype/toFixed/length.js]: while it does not hurt leaving es6id, the current appropriate tag is esid as in this example: https://github.com/tc39/test262/pull/824/files#diff-971804c3f7e390b76d945b7285de923aR5

**Comment by mzgoddard** [comment]
@leobalter is https://github.com/tc39/test262/pull/826/commits/a10a7014176de75b5b8069bf752a53906207db81 good?

**Comment by rwaldron** [comment]
Note to other maintainers: @mzgoddard is employed by Bocoup, which is an Ecma member company.

**Comment by leobalter** [comment]
Sure it is! Thanks!

**Pull request merged**